src/coreclr/vm/jitinterface.cpp

```diff
@@ -7210,7 +7232,8 @@ bool getILIntrinsicImplementationForRuntimeHelpers(MethodDesc * ftn,
             || methodTable == CoreLibBinder::GetClass(CLASS__INTPTR)
             || methodTable == CoreLibBinder::GetClass(CLASS__UINTPTR)
             || methodTable == CoreLibBinder::GetClass(CLASS__RUNE)
-            || methodTable->IsEnum())
+            || methodTable->IsEnum()
```

Comment:
Is this problematic for that case (is it Mono only?) where an enum can be backed by a `float` or `double`?

Reply:
https://github.com/dotnet/runtime/blob/ad8debf62d9ae538c4e1a2f473a5a86377eabbc8/src/libraries/System.Private.CoreLib/src/System/Enum.cs#L12-L22